Rakefile

```diff
@@ -7,12 +7,8 @@ RSpec::Core::RakeTask.new(:spec) do |task|
   task.rspec_opts = '--color'
 end
 
-task :build do
-  sh 'bundle exec jekyll build --config _config.yml,_config_en.yml --trace'
-end
-
 task :buildboth do
-  sh 'bundle exec jekyll build --trace'
+  sh 'bundle exec jekyll build --config _config.yml,_config_en.yml --trace'
```

Comment:
It's doable, and we intend to do it cleaner in another PR, but that would mean changes to the config files (both Rake and _config files). 

This change here was to remove tasks that didn't work or was redundant.